### ISSUE TITLE
baggageclaim v1.4.0 incompatible with linux v5

### ISSUE THREAD

**Issue by pivotal-jamie-klassen** [concourse/concourse#5241]
concourse/concourse#4964 affects 5.2.x

either we could switch our testflight job to run on a linux-4.x worker (which would probably need to be managed outside GKE, which only seems to allow the latest node images), or we could create a new baggageclaim release that only includes the relevant changes from concourse/baggageclaim#35 to bring back linux 5 compatibility.

**Comment by pivotal-jamie-klassen** [comment]
We're willing to assume some risk and continue running our release pipelines on Linux v5 and pending the relevant testflight test. Linux has, in general, very strong backwards compatibility guarantees - the reason we encountered this issue has much more to do with our anti-idiomatic use of overlay. If that hasn't caused bugs in production scenarios so far, it seems imprudent to start making changes.

tl;dr - if it ain't broke, don't fix it.
